backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/bottomnav/BpkBottomNav.kt

```diff
@@ -74,51 +90,108 @@ fun BpkBottomNav(
-    elevation: Dp = BottomNavigationDefaults.Elevation,
+    elevation: Dp = BpkElevation.Lg,
 ) {
-    BottomNavigation(
-        modifier = modifier,
-        backgroundColor = BpkTheme.colors.surfaceDefault,
+    Surface(
+        color = BpkTheme.colors.surfaceDefault,
         contentColor = BpkTheme.colors.textSecondary,
-        elevation = elevation,
+        shadowElevation = elevation,
+        modifier = modifier,
     ) {
-        items.forEach { tabItem ->
-            BottomNavigationItem(
-                selected = selectedItemId == tabItem.id,
-                onClick = { onTabClicked(tabItem.id) },
-                icon = {
-                    Box {
-                        when (tabItem) {
-                            is IconBottomNavItem -> BpkIcon(
-                                icon = tabItem.icon,
-                                contentDescription = null,
-                                size = BpkIconSize.Large,
-                            )
-                            is PainterBottomNavItem -> Icon(
-                                modifier = Modifier.height(BpkSpacing.Lg),
-                                painter = tabItem.painter,
-                                contentDescription = null,
-                            )
-                        }
-                        if (tabItem.showBadge) {
-                            NotificationDot(
-                                Modifier
-                                    .align(Alignment.BottomEnd)
-                                    .offset(x = 1.dp, y = (-2).dp),
-                            )
-                        }
-                    }
-                },
-                label = {
-                    BpkText(
-                        text = tabItem.title,
-                        style = BpkTheme.typography.label3,
-                        maxLines = 1,
-                        overflow = TextOverflow.Ellipsis,
-                    )
-                },
-                selectedContentColor = BpkTheme.colors.textLink,
-                unselectedContentColor = BpkTheme.colors.textSecondary,
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .height(56.dp)
+                .selectableGroup(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+        ) {
+            items.forEach { tabItem ->
+                BottomNavigationItem(
+                    tabItem = tabItem,
+                    selected = selectedItemId == tabItem.id,
+                    onClick = { onTabClicked(tabItem.id) },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun RowScope.BottomNavigationItem(
+    tabItem: BpkBottomNavItem,
+    selected: Boolean,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+
+    val ripple = rememberRipple(bounded = false, color = BpkTheme.colors.textLink)
+
+    val contentColor by animateColorAsState(
+        label = "BottomNavItem content color",
```

Comment:
what does the label do?

Reply:
That label is for the animation preview tool in AndroidStudio. You can find the animation you want and debug it there. There was a new lint rule added about this label, so I updated the code.